iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergMetaHook.java

```diff
@@ -159,11 +162,16 @@ public void commitCreateTable(org.apache.hadoop.hive.metastore.api.Table hmsTabl
 
   @Override
   public void preDropTable(org.apache.hadoop.hive.metastore.api.Table hmsTable) {
+    // do nothing
```

Comment:
When does this version of the hook get called (vs the one with deleteData param)?